Detections/SecurityAlert/AVSpringShell.yaml

```diff
@@ -0,0 +1,44 @@
+id: 3bd33158-3f0b-47e3-a50f-7c20a1b88038
+name: AV detections related to SpringShell Vulnerability
+description: |
+  'This query looks for Microsoft Defender AV detections related to SpringShell Vulnerability. In Microsoft Sentinel the SecurityAlerts table includes only the Device Name of the affected device, 
+   this query joins the DeviceInfo table to clearly connect other information such as Device group, ip,logged on users etc. This way, the Microsoft Sentinel user can have all the pertinent device info in one view for all the alerts.
+   Refrence: https://www.microsoft.com/security/blog/2022/04/04/springshell-rce-vulnerability-guidance-for-protecting-against-and-detecting-cve-2022-22965/'
+severity: High
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - SecurityAlert
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+tags:
+  - CVE-2022-22965
+  - SpringShell 
+  - Spring4Shell
+query: |
+  let SpringShell_threats = dynamic(["SpringShell"]);
+  DeviceInfo
+  | extend DeviceName = tolower(DeviceName)
+  | join ( SecurityAlert
+  | where ProviderName == "MDATP"
+  | extend ThreatFamilyName = tostring(parse_json(ExtendedProperties).ThreatFamilyName)
+  | where ThreatFamilyName in (SpringShell_threats)
+  | extend CompromisedEntity = tolower(CompromisedEntity)
+  ) on $left.DeviceName == $right.CompromisedEntity
```

Comment:
the schema is over 50 columns for this, wondering if we should be doing any kind of summarize or project to reduce output fields?